.ci/scripts/go-plus/vcr-cassette-update/vcr_cassette_update.sh

```diff
@@ -127,7 +127,7 @@ if [[ -n $FAILED_TESTS_PATTERN ]]; then
   done
 
   # store cassettes
-  gsutil -m -q cp fixtures/* gs://vcr-nightly/beta/$today/$build_id/cassettes/fixtures/
+  gsutil -m -q cp fixtures/* gs://ci-vcr-cassettes/beta/fixtures/
```

Comment:
Are we changing the bucket of the cassettes? 

I need to make the the new service account for the downstream builder has access to this new bucket.

cloudbuild-downstream-builder@graphite-docker-images.iam.gserviceaccount.com

Reply:
Ah sorry I saw vcr-nightly -> ci-vcr-cassette. I see you're aligning to the old bucket. Nevermind!
https://github.com/GoogleCloudPlatform/magic-modules/pull/9678/files/40febc45f62fa994686e8504fa33c111bbd6bcaf..84b341666750248fbd6fd7c49e958a3aa9443b47#diff-b68cddea7a32f9e31e9e1d692d93e172563dc5c8182d2a312d04c45bdd7e685dR130